services/QuillLMS/client/app/bundles/Staff/components/comprehension/activitySessions/promptTable.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from "react";
+import stripHtml from "string-strip-html";
+
+import { DataTable, Spinner } from '../../../../Shared/index';
+import { PROMPT_ATTEMPTS_FEEDBACK_LABELS, PROMPT_HEADER_LABELS } from '../../../../../constants/comprehension';
+import { ActivityInterface, PromptInterface } from "../../../interfaces/comprehensionInterfaces";
+
+interface PromptTableProps {
+  activity: ActivityInterface;
+  prompt: PromptInterface;
+  showHeader?: boolean;
+}
+const PromptTable = ({ activity, prompt, showHeader }: PromptTableProps) => {
+
+
+  function formatFirstTableData(prompt: any) {
+    const { attempts } = prompt;
+    const keys = attempts && Object.keys(attempts);
+    let completed: boolean;
+    if(keys.length === 5) {
```

Comment:
We should probably make the `5` here into a const so it's easier to find if we change the number of attempts at some point in the future.

Reply:
Also, even though you're nullchecking `attempts` above, if `keys` is null this line is going to error.

Reply:
good point, I already have a const for this